TABTestKit/Classes/Elements/Attributes.swift

```diff
@@ -10,23 +10,49 @@ import Foundation
 /// Serves as the namespace for Element, since you can't nest enums in a protocol.
 public enum ElementAttributes {
 	
+	/// Represents a state that an element can be in.
+	///
+	/// - exists: The element exists, but may not necessarily be visible or hittable. It's often a good idea to check this state first to avoid flakiness.
+	/// - hittable: The element is hittable, i.e. can be tapped.
+	/// - visible: The element is visible in its parent.
+	/// - visibleIn: The element is visible in the provided element.
+	/// - selected: The element is selected.
+	/// - enabled: The element is enabled.
+	/// - hasKeyboardFocus: The element has keyboard focus (accepting keyboard entry or user input)
 	public enum State {
 		
 		case exists
 		case hittable
 		case visible
+		case visibleIn(Element)
```

Comment:
Since `visible` uses the parent by default, that might not be what people want, so this adds `visibleIn` which requires the dev to pass in the element they want it to be visible in.